src/app/page.tsx

```diff
@@ -3,14 +3,15 @@ import { AboutSection } from "@/components/AboutSection";
 // import Image from "next/image";
 // import NavBar from "@/components/NavBar";
 import { GoogleGeminiEffectDemo } from "@/components/GoogleGeminiEffectDemo";
-
+import { FeaturesSection } from "@/components/ui/FeaturesSection"; // Import the Features Section
```

Comment:
comment out this import from here, i'll uncomment it once im done testing the component, i'll uncomment it afterwards